free-courses-en.md

```diff
@@ -281,6 +281,7 @@
 * [Princeton Algorithms, Part 1](https://www.coursera.org/course/algs4partI)
 * [Problem Solving With Java](https://www.udacity.com/course/intro-to-java-programming--cs046) (Udacity)
 * [Spring 5 Core - An Ultimate Guide](https://www.udemy.com/learn-spring-5-core-from-scratch/) - Somnath Musib (Udemy)
+* [Java 3 Month Program](https://www.youtube.com/watch?v=4hwmRUup7BE&list=PL7LtGuh_nbfphuB58q2SQ7C9k52THkRoN) - Bright Boost (Youtube)
```

Comment:
Could you please use a link to the playlist endpoint instead of watch? This will put the user on the playlist view rather than throw them into a video already.

Could you also please write YouTube as "YouTube", not "Youtube" as this is the spelling used on the site.

```suggestion
* [Java 3 Month Program](https://www.youtube.com/playlist?list=PL7LtGuh_nbfphuB58q2SQ7C9k52THkRoN) - Bright Boost (YouTube)
```